src/components/ButtonPanel.js

```diff
@@ -4,35 +4,35 @@ import { Button } from './Button';
 export class ButtonPanel extends React.Component {
   render() {
     return (
-      <div>
-        <div>
-          <Button id='AC' />
-          <Button id='+/-' />
-          <Button id='%' />
-          <Button id='/' />
+      <div id='panel'>
+        <div className='row'>
+          <Button name='AC' />
+          <Button name='+/-' />
+          <Button name='%' />
```

Comment:
Hi let's send the colour props to the grey buttons instead so the others can use the default.